windows/bleachbit_lang.nsh

```diff
@@ -0,0 +1,1810 @@
+	LangString BLEACHBIT_COMPONENT_AUTOSTART_DESCRIPTION ${LANG_GEORGIAN} "Creates Autostart Shortcut for BleachBit"
+	LangString BLEACHBIT_COMPONENT_TRANSLATIONS_TITLE ${LANG_GEORGIAN} "Translations"
+	LangString BLEACHBIT_COMPONENT_TRANSLATIONS_DESCRIPTION ${LANG_GEORGIAN} "BleachBit Translations"
+	LangString BLEACHBIT_COMPONENT_INTEGRATESHRED_TITLE ${LANG_GEORGIAN} "Integrate Shred"
+	LangString BLEACHBIT_COMPONENT_INTEGRATESHRED_DESCRIPTION ${LANG_GEORGIAN} "Integrate BleachBit's Shred into Windows Explorer"
+	LangString BLEACHBIT_SHELL_TITLE ${LANG_GEORGIAN} "Shred with BleachBit"
+	LangString BLEACHBIT_MUI_FINISHPAGE_LINK ${LANG_GEORGIAN} "Visit the ${prodname} web site"
+	LangString BLEACHBIT_UNINSTALLTEXT ${LANG_GEORGIAN} "BleachBit will be uninstalled from the following folder. Click 'Uninstall' to start the uninstallation. WARNING: The uninstaller completely removes the installation directory including any files (such as custom cleaners) that you may have added or changed!"
+!endif
+
+!ifdef LANG_GERMAN
+	; Translation 2019-04-01 - 2019-04-02 by https://github.com/Tobias-B-Besemer
+	LangString BLEACHBIT_UPGRADE_UNINSTALL ${LANG_GERMAN} "${prodname} ist bereits installiert. Klicke 'OK' zum Deinstallieren der alten Version vor dem Upgrade, oder klicke 'Abbrechen' zum Abbruch des Upgrade."
+	LangString BLEACHBIT_COMPONENT_CORE_TITLE ${LANG_GERMAN} "Kern"
+	LangString BLEACHBIT_COMPONENT_CORE_DESCRIPTION ${LANG_GERMAN} "Installiert den BleachBit Kern"
+	LangString BLEACHBIT_COMPONENTGROUP_SHORTCUTS_TITLE ${LANG_GERMAN} "Verknüpfungen"
+	LangString BLEACHBIT_COMPONENTGROUP_SHORTCUTS_DESCRIPTION ${LANG_GERMAN} "Erzeugt Verknüpfungen für BleachBit"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_TITLE ${LANG_GERMAN} "Start Menü"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_DESCRIPTION ${LANG_GERMAN} "Erzeugt 'Start Menü'-Verknüpfung für BleachBit"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_NO_UAC ${LANG_GERMAN} "Keine Benutzerkontensteuerung"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_DEBUGGING_TERMINAL ${LANG_GERMAN} "Debugging-Terminal"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_HOME_PAGE ${LANG_GERMAN} "Homepage"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_UNINSTALL ${LANG_GERMAN} "Uninstaller"
+	LangString BLEACHBIT_COMPONENT_DESKTOP_TITLE ${LANG_GERMAN} "Desktop"
+	LangString BLEACHBIT_COMPONENT_DESKTOP_DESCRIPTION ${LANG_GERMAN} "Erzeugt Desktop-Verknüpfung für BleachBit"
+	LangString BLEACHBIT_COMPONENT_QUICKLAUNCH_TITLE ${LANG_GERMAN} "Schnellstart"
+	LangString BLEACHBIT_COMPONENT_QUICKLAUNCH_DESCRIPTION ${LANG_GERMAN} "Erzeugt Schnellstart- Verknüpfung für BleachBit"
+	LangString BLEACHBIT_COMPONENT_AUTOSTART_TITLE ${LANG_GERMAN} "Autostart"
+	LangString BLEACHBIT_COMPONENT_AUTOSTART_DESCRIPTION ${LANG_GERMAN} "Erzeugt Autostart-Verknüpfung für BleachBit"
+	LangString BLEACHBIT_COMPONENT_TRANSLATIONS_TITLE ${LANG_GERMAN} "Übersetzungen"
+	LangString BLEACHBIT_COMPONENT_TRANSLATIONS_DESCRIPTION ${LANG_GERMAN} "Ble
```

Comment:
Like my other comment, let's please include only the languages in the NSIS installer for which the BleachBit application has decent translations.

Reply:
I got equal with NsisMultiUserLang.nsh. But if you let the NSH files be auto-generated, I will change it again. But in the next PR! I will still need that data for #529 and the other languages are now just comments...

Reply:
Addition: That's my German translation you pointed to! ^^
Language Strings not used in MUI_Language doesn't get compiled!